packages/frontend/src/system-integration/webxdc.ts

```diff
@@ -6,18 +6,30 @@ import { BackendRemote, Type } from '../backend-com'
 import { runtime } from '@deltachat-desktop/runtime-interface'
 
 export function initWebxdc() {
-  BackendRemote.on('WebxdcStatusUpdate', (accountId, { msgId }) => {
-    runtime.notifyWebxdcStatusUpdate(accountId, msgId)
-  })
-  BackendRemote.on('WebxdcRealtimeData', (accountId, { msgId, data }) => {
-    runtime.notifyWebxdcRealtimeData(accountId, msgId, data)
-  })
-  BackendRemote.on('MsgsChanged', (accountId, { msgId }) => {
-    runtime.notifyWebxdcMessageChanged(accountId, msgId)
-  })
-  BackendRemote.on('WebxdcInstanceDeleted', (accountId, { msgId }) => {
-    runtime.notifyWebxdcInstanceDeleted(accountId, msgId)
-  })
+  BackendRemote.on(
+    'WebxdcStatusUpdate',
+    (accountId: number, { msgId }: { msgId: number }) => {
+      runtime.notifyWebxdcStatusUpdate(accountId, msgId)
+    }
+  )
+  BackendRemote.on(
+    'WebxdcRealtimeData',
+    (accountId: number, { msgId, data }: { msgId: number; data: any }) => {
+      runtime.notifyWebxdcRealtimeData(accountId, msgId, data)
+    }
+  )
+  BackendRemote.on(
+    'MsgsChanged',
+    (accountId: number, { msgId }: { msgId: number }) => {
+      runtime.notifyWebxdcMessageChanged(accountId, msgId)
+    }
+  )
+  BackendRemote.on(
+    'WebxdcInstanceDeleted',
+    (accountId: number, { msgId }: { msgId: number }) => {
+      runtime.notifyWebxdcInstanceDeleted(accountId, msgId)
+    }
+  )
```

Comment:
What is this about? Why do we have to convert `data` to `any`?

Reply:
Hmm, looks as if linting in my VSCode was broken yesterday night and tricked me to add types. Just removed it again.